atlas/packages/app/src/utils/number.ts

```diff
@@ -1,3 +1,18 @@
 export const formatNumber = (num: number): string => {
   return num.toLocaleString('en-US').split(',').join(' ')
 }
+
+export const formatNumberShort = (num: number): string => {
```

Comment:
There is a tiny inconsistency here, this returns `3.0K` for `3001` and `3K` for `3000`, same things happens with milions

Reply:
Good catch! Changed the function to always skip `.0` suffix